sdk/file_sdk/OPCFileFormat/test/custom_properties_tests.cpp

```diff
@@ -88,3 +89,76 @@ TEST(CustomProperties, GetProperties_WrongPropertySubNode_NodeSkipped) {
   EXPECT_EQ(tags[0], CustomProperty("PropName0", "PropValue0"));
   EXPECT_EQ(tags[1], CustomProperty("PropName2", "2"));
 }
+
+TEST(CustomProperties, UpdateProperties_Delete_MultipleProperties_MultipleTags) {
```

Comment:
>Delete_MultipleProperties_MultipleTags [](start = 40, length = 38)

Test does not describe very well what it tests.
looks to me more like:
DeleteMultipleProperties_OnlyUndeletedPropertiesLeft